pkg/controllers/allocation/binpacking/packable.go

```diff
@@ -164,16 +165,24 @@ func (p *Packable) validateArchitecture(schedule *scheduling.Schedule) error {
 	return nil
 }
 
-func (p *Packable) validateOperatingSystem(schedule *scheduling.Schedule) error {
-	if schedule.Requirements.OperatingSystems().Intersection(p.OperatingSystems()).Len() == 0 {
-		return fmt.Errorf("operating system %s is not in %v", p.OperatingSystems(), schedule.Requirements.OperatingSystems().List())
+func (p *Packable) validateZones(schedule *scheduling.Schedule) error {
+	packableZones := sets.String{}
```

Comment:
Optional:

Not sure if I mentioned earlier, but go style would be to let the "packable" piece of this variable be inferred by context of it being within a Packable struct. This is known as "stuttering". Sometimes if you have multiple concepts that collide (i.e. two different sources of zone), you need to get a bit clever about naming.

Reply:
This is I think actually an artifact of me doing something stupidly complicated here where I had two sets of zones and had to differentiate them. Fixed.